flink-runtime/src/main/java/org/apache/flink/runtime/minicluster/MiniCluster.java

```diff
@@ -561,6 +561,7 @@ private void setupDispatcherResourceManagerComponents(
                         metricRegistry,
                         new MemoryExecutionGraphInfoStore(),
                         metricQueryServiceRetriever,
+                        Collections.emptySet(),
```

Comment:
What's the reason for not supporting the minicluster?

Reply:
Will be done as part of [FLINK-31895](https://issues.apache.org/jira/browse/FLINK-31895)